source/plugin/blocks/tileentities.rst

```diff
@@ -66,21 +67,20 @@ line, the second attempts to set it and returns the boolean value indicating its
     public Optional<Text> getFirstLine(TileEntity entity) {
         Optional<SignData> data = entity.getOrCreate(SignData.class);
         if (data.isPresent()) {
-            return Optional.of(data.get().getLine(0));
-        } else {
-            return Optional.empty();
+            return Optional.
+                of(data.get().getValue(Keys.SIGN_LINES).get().get(0));
```

Comment:
use [lines()](https://jd.spongepowered.org/org/spongepowered/api/data/manipulator/mutable/tileentity/SignData.html#lines--) instead of `getValue(Keys.SIGN_LINES)` here